.gitignore

```
@@ -8,3 +8,4 @@ polkadot_argument_parsing
 **/chains/
 *.iml
 .env
```

is this `.env` just a helper or required to run the tests?
If so, I would suggest the following and provide an example.

```suggestion
.env
!.env-example
```

---

I dind't add the `.env`, only `bin`

---

The question still stands :)
If the `.env` is not mandatory, we can skip this change.